Update azure.md (#1494)
rancher/rancher-docs: docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-cloud-providers/azure.md

   <details id="v2.6.0-cloud-provider-config-file">
     <summary>Example Cloud Provider Config</summary>

> ```yaml

Reviewer: What's the expected rendered output here? This block doesn't seem to have the same missing indentation as the other blocks. 

The [build is also failing](https://github.com/rancher/rancher-docs/actions/runs/11133756480/job/30940522383?pr=1494) without the existing indentation.

1. Select **External** from the **Cloud Provider** drop-down in the **Cluster Configuration** section.

> :::note
> When setting **Cloud Provider** to **External**, all nodes will automatically be tainted with node.cloudprovider.kubernetes.io/uninitialized=true until the Cloud Provder is installed. It is recommended to deploy the CPI as an add-on job as described below.
> :::

Reviewer: ```suggestion
  :::note

  When setting **Cloud Provider** to **External**, all nodes are automatically tainted with `node.cloudprovider.kubernetes.io/uninitialized=true` until the Cloud Provider is installed. It is recommended to deploy the CPI as an add-on job as described below.

  :::
```

Suggestion updates the text to present tense and adds indentation so the note aligns with the text of step 1.
